src/platform/routes/StudentPage/components/Footer/index.module.scss

```diff
@@ -0,0 +1,37 @@
+@import "~assets/styles/_global";
```

Comment:
for underscored sass files "_global.scss", no need to use the underscore on the import.

Reply:
Done